documentation/adr/0002_replace_google_sheets_with_big_query.md

```diff
@@ -0,0 +1,14 @@
+# Replace Google Sheets with Big Query as a reporting database
+Date: 20/01/2020
+
+## Status
+approved
+
+## Context
+We need to replace google sheets as a reporting database due to us approaching the spreadsheet limit.
+
+## Decision
+To use Google Big Query instead of Google Sheets and export our data into Big Query tables.
```

Comment:
Should we also add some notes here about the options we explored that were *not* Big Query, and why we went with Big Query instead of them?